share/spice/news/news.js

```diff
@@ -77,22 +77,24 @@
 
         var searchTerm = DDG.get_query().replace(/(?: news|news ?)/i, '').trim();
 
-        if (api_result.error || goodStories >= 3) {
+        if (api_result.error || goodStories < 3) {
```

Comment:
Hmmm, I was just copying what was there. Unclear about that...maybe we should just do an `!api_result` guard? Will investigate and get back.